### PR TITLE
docs: fix 404 links

### PR DESCRIPTION
**Pull request by CoreyCole**
These links in the README were 404ing. If those projects have been deleted/discontinued we should remove them (this PR)

If they have moved/merged into different projects, let me know I can update the links instead of removing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,6 @@
 
 * Flutter: [FlutterFire plugins](https://github.com/flutter/plugins/blob/master/FlutterFire.md)
 
-* Node (via dart2js): [Anatoly Pulyaevskiy](https://github.com/pulyaevskiy) has
-  been working on unofficial wrappers.
-  * [package:firebase_admin_interop](https://pub.dev/packages/packages/firebase_admin_interop)
```
Comment (kevmoo) [inline README.md]: Would you fix them instead?

https://pub.dev/packages/firebase_admin_interop etc etc

**Comment by kevmoo** [comment]
While you're at it, want to fix the homepage link in `pubspec.yaml`?

File: README.md (hunk 1 above)
Comment (CoreyCole) [reply README.md]: Done! But I don't see an interop package for cloud firestore: https://pub.dev/packages?q=firestore_interop

Comment (CoreyCole) [reply README.md]: I found the repository, but maybe the package was taken down https://github.com/pulyaevskiy/firestore-interop

**Comment by kevmoo** [comment]
https://pub.dev/packages/firestore_interop – not taken down, just not
updated.

Delete this one! Thanks!

**Comment by CoreyCole** [comment]
@kevmoo I updated the hompage link and also incremented the version to `8.0.0-dev+1` in pubspec.yaml

**Pull request merged**